detecto/core.py

```diff
@@ -252,7 +252,12 @@ def __init__(self, classes=None, device=None, pretrained=True):
         self._device = device if device else config['default_device']
 
         # Load a model pre-trained on COCO
-        self._model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=pretrained)
+        if modelname == "fasterrcnn_resnet50_fpn":
+            self._model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=pretrained)
+        elif modelname == "fasterrcnn_mobilenet_v3_large_fpn":
+            self._model = torchvision.models.detection.fasterrcnn_mobilenet_v3_large_fpn(pretrained=pretrained)
+        elif modelname == "fasterrcnn_mobilenet_v3_large_320_fpn":
+            self._model = torchvision.models.detection.fasterrcnn_mobilenet_v3_large_320_fpn(pretrained=pretrained)
```

Comment:
suggestion: if none of the strings match, throw a ValueError explaining the error